dotnet/src/VectorDataIntegrationTests/SqliteIntegrationTests/Support/SqliteFixture.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using VectorDataSpecificationTests.Support;
+
+namespace SqliteIntegrationTests.Support;
+
+public class SqliteFixture : VectorStoreFixture
```

Comment:
Do you think these "generic" provider-specific fixtures improve things? The problem is that there's also VectorStoreCollectionFixture which directly extends VectorStoreFixture, so to follow this pattern you'd have to also have SqliteVectorStoreCollectionFixture which extends that and just overrides TestStore (because no multiple inheritance)...

At the end of the day I'm not sure it adds much value beyond each **concrete** fixture (SqliteBasicFilterTests.Fixture) overriding TestStore directly (as before this PR) - it's almost the same as inheriting SqliteVectorStoreCollectionFixture.

Reply:
OK, no big deal either way.